src/wasm/lsp.rs

```diff
@@ -66,6 +67,8 @@ impl Default for Lsp {
         #[cfg(feature = "console_error_panic_hook")]
         console_error_panic_hook::set_once();
 
+        wasm_logger::init(wasm_logger::Config::default());
```

Comment:
> The result is that the use of an lib (consuming `log`) will error.

1. Does that mean anywhere in our code that used `log::error!()` will error? 
2. Does this line allow the use of `log::error!()` to not error?

Reply:
The `log` dependency provides the ability to implement our own logger as [per here](https://docs.rs/log/0.4.17/log/#implementing-a-logger). The init() call then sets the custom logger as the global `static LOGGER`.

Other logging libraries can then build on top of `log` and implement their own logger...and then use an init() method that sets the logger. (For example, here is wasm-logger [setting the logger](https://gitlab.com/limira-rs/wasm-logger/-/blob/master/src/lib.rs#L173).) This means that after the init() is called, the `log::info!()` will then map the set logger.

Reply:
Since the flux-lsp repo is created in such a way to be independent of wasm (a.k.a. wasm is only one of the targets), this PR only init() the logger when a wasm LspServer is created.

In a follow up PR, I would like to do the same thing for our wasm utils methods (a.k.a. parsing variables) used outside of the server. But was planning to do that separately...since debating about how best to go about this one. (e.g. considering making the utils their own submodule, if it means a smaller than 3.4MB sized wasm fetched. Plan to play with that during any free-form holiday time.)

Reply:
Thanks for explaining it!